src/basic/efivars.h

```diff
@@ -69,6 +69,10 @@ static inline bool is_efi_secure_boot_setup_mode(void) {
         return false;
 }
 
+static inline int cache_efi_options_variable(void) {
+        return -ENOTSUP;
```

Comment:
ENOTSUP is a a weird Linux alias for EOPNOTSUPP. We genreally use the latter, see for example a few lines up. Can you use EOPNOTSUPP here too?

(it's actually even weirder, ENOTSUP in the kernel is different from EOPNOTSUPP, only in userspace it's the same, and the kernel's internal ENOTSUP is converted to EOPNOTSUPP before things are handed to userspace. All in all one never wants to use ENOTSUP)

Reply:
Sure thing, fixed